go/vt/sqlparser/ast.go

```diff
@@ -2207,6 +2207,14 @@ type (
 		alternative  Expr // this is what will be used to Format this struct
 	}
 
+	// JSONUtilityExpr represents set of JSON utility functions
+	// More information on https://dev.mysql.com/doc/refman/8.0/en/json-utility-functions.html
+	JSONUtilityExpr struct {
+		Name      ColIdent
+		StringArg Expr
+		Column    *ColName
```

Comment:
I do not see the Column field being used in the `sql.y` file. Where is it being used?

Reply:
Removed this.